amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,963 @@ input AMPLIFY {
 # https://docs.amplify.aws/cli-legacy/graphql-transformer/overview/#api-category-project-structure
 # but I never could get it to work
 
+"""
+Input data for fetching a token's information from _everywhere_ ??? I'm not sure what that means ???.
+"""
 input TokenFromEverywhereArguments {
+  """Address of the token on the blockchain."""
   tokenAddress: String!
 }
 
+"""
+Input data for fetching the list of members for a specific Colony.
+"""
 input MembersForColonyInput {
+  """Address of the Colony."""
   colonyAddress: String!
+  """Root hash for the reputation state."""
   rootHash: String
+  """ID of the domain within the Colony."""
   domainId: Int
+  """Sorting method to apply to the member list."""
   sortingMethod: SortingMethod
 }
 
+"""
+Input data for creating a unique user within the Colony Network.
+??? I'm not really sure what unique means in this context, I would like to add an explanation here thought ???
```

Comment:
Renamed because amplify creates `CreateUserInput` already. Canonical way to create a user.